client/webserver/site/src/js/orderutil.ts

```diff
@@ -119,6 +120,29 @@ export function settled (order: Order) {
   }, 0)
 }
 
+/* averageRateString returns a formatting string containing the average rate of
+the matches that have been filled in an order. */
+export function averageRateString (ord: Order): string {
+  const nMatch = ord.matches?.length
+  if (nMatch === 0) return '0'
+  let rateStr = Doc.formatCoinValue(app().conventionalRate(ord.baseID, ord.quoteID, averageRate(ord)))
+  if (nMatch > 1) rateStr = '~ ' + rateStr // "~" only makes sense if the order has more than one match.
```

Comment:
I guess this whole function only makes sense for market orders then? If so, the function name should reflect that fact.